Norma-Portuguesa-405-citação-numérica

```diff
@@ -0,0 +1,522 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" default-locale="pt-PT">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Norma Portuguesa 405 (citação numérica)</title>
+    <title-short>NP 405</title-short>
+    <id>http://www.zotero.org/styles/norma-portuguesa-405-citacao-numerica</id>
```

Comment:
@ritacmg, you still need to change the file name to "norma-portuguesa-405-numerica.csl", and update the style ID and "self" link to match.